vmc/resource_vmc_site_recovery.go

```diff
@@ -72,6 +72,10 @@ func resourceSiteRecovery() *schema.Resource {
 
 func resourceSiteRecoveryCreate(d *schema.ResourceData, m interface{}) error {
 
+	err := (m.(*ConnectorWrapper)).authenticate()
+	if err != nil {
+		return fmt.Errorf("Error authenticating in CSP: %s", err)
```

Comment:
Is this error displayed on console during apply? If yes, does CSP make sense to users?

Reply:
Yes this error does get displayed during apply. This is a CSP API and although I did not work on this error message earlier I used the same one from https://github.com/terraform-providers/terraform-provider-vmc/blob/master/vmc/resource_vmc_sddc.go#L277 .  I can change it everywhere to Cloud Server Provider to be consistent in the code.

Reply:
Done